### PR TITLE
Move to balenaCI

### PR DESCRIPTION
**Pull request by nazrhom**
Change-type: patch
Signed-off-by: Robert Günzler <robertg@balena.io>

Connects-to: https://github.com/balena-io/resin-concourse/pull/408
Connects-to: https://github.com/balena-io/ci-images/pull/27
Connects-to: https://github.com/balena-io/scripts/pull/63
Connects-to: https://github.com/balena-io-modules/detectorist/pull/27

### CODE REVIEW

**Comment by robertgzr** [comment]
@balena-ci 	retest

**Comment by robertgzr** [comment]
- [ ] update https://github.com/balena-io/ci-images/pull/27
- [ ] add balena-engine plugin to https://github.com/balena-io-modules/detectorist
- [ ] remove reference to wip build image https://github.com/balena-io/balena-concourse/pull/408/commits/c06a30653928a819a71399354a83816333d4b252
- [ ] remove detectorist workaround https://github.com/balena-io/scripts/pull/63/commits/3f6d8e5e4da746c4af261de5534a0fd66fc4acf9

**Comment by robertgzr** [comment]
@balena-ci retest

**Comment by robertgzr** [comment]
@balena-ci retest

**Comment by robertgzr** [comment]
@balena-ci retest

**Comment by robertgzr** [comment]
@balena-ci retest

**Comment by robertgzr** [comment]
@balena-ci retest

File: hack/make/binary-balena [hunk 1]
```diff
@@ -9,8 +9,5 @@ set -e
 	export LDFLAGS="$LDFLAGS"
 	export BUILDFLAGS=( "${BUILDFLAGS[@]/netgo /}" ) # disable netgo, since we don't need it for a dynamic binary
 	source "${MAKEDIR}/.binary"
-
-	for file in balena-engine-daemon balena-engine-containerd balena-engine-containerd-shim balena-engine-containerd-ctr balena-engine-runc balena-engine-proxy; do
-		ln -sf $balena_BINARY_NAME "$DEST/$file"
-	done
+	source "${MAKEDIR}/.binary-symlinks"
```
Comment (roman-mazur) [inline hack/make/binary-balena]: nit: I doubt you really need to call `source` here (the script is not supposed to set new env variables)

Comment (robertgzr) [reply hack/make/binary-balena]: will do `. "${MAKEDIR}/.binary-symlinks"` instead

Comment (roman-mazur) [reply hack/make/binary-balena]: `.` is equivalent to `source`
I was thinking more about `bash "${MAKEDIR}/.binary-symlinks"`

Comment (robertgzr) [reply hack/make/binary-balena]: https://github.com/balena-os/balena-engine/blob/ci/hack/make/binary#L9 that is how upstream does it... although they also do https://github.com/balena-os/balena-engine/blob/ci/hack/test/e2e-run.sh#L71

**Comment by robertgzr** [comment]
@balena-ci retest

**Pull request merged**